frontend/src/Components/InspirationSection/InspirationSection.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useRef, useState } from "react";
+import { CardComp } from "../Card/Card";
+import { Container } from "./InspirationSectionStyle";
+
+interface InspirationSectionProps {
+  //cards as props will be add after connection to db
+}
+type CardInfo = {
+  img: string;
+  title: string;
+  text: string;
+};
+
+export const Inspiration: React.FC<InspirationSectionProps> = ({}) => {
+  const InsparationSection = useRef<HTMLDivElement>(null);
+  const containerRef = useRef<HTMLDivElement>(null);
```

Comment:
Passing `null` as an initial value of a state is like not passing a value at all, so you can just remove it.